pkg/proxy/winkernel/proxier_test.go

```diff
@@ -1,2031 +0,0 @@
-// +build windows
-
```

Comment:
Leaving just the +build line, in a file with no `package` line seems like it would break the Windows build...  Should this file be removed entirely?

Reply:
Yes, we can remove it completely.

I'll look into the plan and get back on the tests.

Reply:
as a side note: it would be really helpful to use `proxier_windows_test.go` instead of build tag comments so that we can trivially trigger the cross build job on files like this.

Reply:
> Leaving just the +build line, in a file with no package line seems like it would break the Windows build... Should this file be removed entirely?

I triggered the cross-build job manually and found it did NOT break the Windows build.

Reply:
BTW we have a PR to fix manual triggering which should be deployed ~ tomorrow morning https://github.com/kubernetes/test-infra/pull/5834